boofuzz/monitors/external_monitor.py

```diff
@@ -1,8 +1,15 @@
-class External:
+from .imonitor import IMonitor
+
+
+class External(IMonitor):
     """
     External instrumentation class
     Monitor a target which doesn't support a debugger, allowing external
-    commands to be called
+    commands to be called.
+
+    .. deprecated:: 1.0
```

Comment:
I guess we can mark this one as deprecated from 0.2.0 on if we don't plan on supporting it any further.